code/game/objects/items/devices/multitool.dm

```diff
@@ -28,77 +28,12 @@
 	var/obj/machinery/buffer // simple machine buffer for device linkage
 	toolspeed = 1
 	usesound = 'sound/weapons/empty.ogg'
-	var/datum/integrated_io/selected_io = null  //functional for integrated circuits.
 	var/mode = 0
 
-/obj/item/multitool/examine(mob/user)

```

Comment:
Keep this check and only remove the `if(selected_io)` lines; the buffer is used for stuff like linking tcomms and quantum pads